### ISSUE TITLE
Page API does not return contentlet json for "Advanced Templates" 

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18158]
If you create a page using an advanced template (e.g. blank template) and add content to it, you do not get the json contentlets back in the page api.  The rendered content comes back but the contentlets json do not come back.

![Screen Shot 2020-03-19 at 9 07 37 AM](https://user-images.githubusercontent.com/934364/77070777-3d85b980-69c1-11ea-8676-8a296079d2f3.png)

**Comment by jdotcms** [comment]
[PR](https://github.com/dotCMS/core/pull/18171 l)

**Comment by wezell** [comment]
We have a customer waiting on this. If it passes internal qa,  I'd like to see it merged into 5.2.8 even without a test

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.8 // Postgress // FF
